### ISSUE TITLE
Support v16

### ISSUE THREAD

**Issue by kuncevic** [ng-select/ng-select#2226]
There is a PR https://github.com/ng-select/ng-select/pull/2224 that brings v16 support.

Anyway to merge that some time soon?

**Comment by gurelsoycaner** [comment]
waiting too

**Comment by ak274** [comment]
As in v16 inputs can be mark as required. IMO `items ` now should be required.

**Comment by steve3d** [comment]
> As in v16 inputs can be mark as required. IMO `items ` now should be required.

I don't think so, There are some use cases only use `ng-option` without any `items`

**Comment by tomer953** [comment]
pr merged, but no release yet to npm.

**Comment by shessafridi** [comment]
pr merged, but no release yet to npm. (2)

**Comment by dgw2jr** [comment]
> pr merged, but no release yet to npm. (2)

Any idea when there will be an npm release?

**Comment by sream** [comment]
Any timeframe for the release?

**Comment by tomer953** [comment]
Guys, it it not helping to spam the same comment over and over. if you want to show support on some opinion, press the thumbs up button instead. 
let the author's do theirs work

**Comment by terencehonles** [comment]
@varnastadeus thanks for merging #2224, but can we get the package released to NPM, and can we please **also** have @ng-select/ng-option-highlight published too? It's been missed in a lot of the recent releases.

Does it make sense to set up any sort of release automation that can build and publish to NPM based on a tag or creating the release in GitHub's UI? Does it also make sense to expand the set of maintainers so that this is not only your chor?

Thanks in advance!

**Comment by harvanchik** [comment]
The version guide in the ReadMe shows that v16+ maps to ng-select v11, so I imagine that the package will be released to NPM any day now.

**Comment by cankarakoyun** [comment]
This PR is fixed NPM deployment fail: https://github.com/ng-select/ng-select/pull/2234 Some who has permission can merge this?

**Comment by sream** [comment]
Since the main maintainers are not responding I've started to build the library internally, but it looks like it is barely maintained since many dependencies are not up-to-date. I've already set to deprecate it from our project because this is a no-go, also to wait three weeks for a dependencies bump is barely acceptable since the community has no control over the process. This was the last library to block our update to Angular 16.

Here is a list of issues that yarn logged:
`warning " > @ng-bootstrap/ng-bootstrap@13.1.1" has incorrect peer dependency "@angular/common@^14.1.0".`
`warning " > @ng-bootstrap/ng-bootstrap@13.1.1" has incorrect peer dependency "@angular/core@^14.1.0".`
`warning " > @ng-bootstrap/ng-bootstrap@13.1.1" has incorrect peer dependency "@angular/forms@^14.1.0".`
`warning " > @ng-bootstrap/ng-bootstrap@13.1.1" has incorrect peer dependency "@angular/localize@^14.1.0".`
`warning " > @angular-eslint/schematics@14.4.0" has incorrect peer dependency "@angular/cli@>= 14.0.0 < 15.0.0".`

I've managed to build and publish it in the end with the instructions from readme but be aware to adapt release.sh to your needs if you go down this path since the script is doing various things.

**Comment by jeroenvheel** [comment]
@ng-bootstrap/ng-bootstrap v15 is now released with support for Angular v16

**Comment by cankarakoyun** [comment]
> @ng-bootstrap/ng-bootstrap v15 is now released with support for Angular v16
I checked npm but there is still version look 10.0.4
